modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogTableDescriptor.java

```diff
@@ -47,6 +48,8 @@ public class CatalogTableDescriptor extends CatalogObjectDescriptor {
 
     private final int tableVersion;
```

Comment:
probably, it would be better to remove `tableVersion` at all, rename `CatalogTableDescriptor#tableVersion` to `latestVersion` and delegate call to `schemaVersions`, WDYT?

Reply:
I think it's a good idea